common/views/components/ApiToolbar/ApiToolbar.tsx

```diff
@@ -37,26 +38,78 @@ const includes = [
  * from the catalogue.
  *
  */
-async function createTzitzitLink(imageId: string): Promise<Prop> {
-  const apiUrl = `https://api.wellcomecollection.org/catalogue/v2/images/${imageId}?include=source.contributors`;
-  const image: Image = await fetch(apiUrl).then(res => res.json());
+
+function setTzitzitParams(data: {
+  title: string;
+  sourceLink: string;
+  licence: string;
+  author?: string;
+}): string {
+  // If license is INC, return as the tool should not be made available
```

Comment:
Can we be a bit more explicit here? "we should not be using in copyright images in Stories"

Reply:
I'll change it! While we're at it, I'll confirm with you as well, this change will apply to the existing functionality; so images that would've had the link before won't have it anymore.

Reply:
Yep, that makes sense.